src/pudl/transform/eia930.py

```diff
@@ -19,7 +21,7 @@
     compute_kind="pandas",
 )
 def core_eia930__hourly_operations_assets(
-    raw_eia930__balance: pd.DataFrame,
+    raw_eia930__balance: Path,
```

Comment:
This will do in a pinch, but I think in an ideal world an IOManager would handle reading from that path, so that `raw_eia930__*` assets are data frames just like the other `raw_*__*` assets.